plugins/duo_admin/komand_duo_admin/tasks/monitor_logs/task.py

```diff
@@ -376,7 +376,10 @@ def get_trust_monitor_event(self, mintime: int, maxtime: int, next_page_params:
         )
         self.logger.info(f"Parameters for get trust monitor events set to {parameters}")
         response = self.connection.admin_api.get_trust_monitor_events(parameters).get("response", {})
-        self.logger.info(f"Response returned from get trust monitor events: {response}")
+        self.logger.info(
+            f"Parameters to return from get admin logs set to {parameters}. "
```

Comment:
Is this correct - should it not refer to trust monitor events?

Reply:
You're right, thanks for spotting